migrations/Version20201124111815.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20201124111815 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Create user table';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        $this->addSql('CREATE TABLE sf_user (id VARCHAR(256) NOT NULL, roles LONGTEXT NOT NULL COMMENT \'(DC2Type:json)\', enabled TINYINT(1) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
```

Comment:
Why the `sf_` prefix in the table name?

Reply:
Because `user` is a reserved word in many RDBMs like postgresql. Symfony [recommends not to use it](https://symfony.com/doc/current/doctrine.html#creating-an-entity-class) (see at the end of the section). Doctrine recommends to quote the name, anyway I had issues at the past even with quotes. Some other bundles use a prefix to avoid this and I started to do the same. 

If you don't like it, we can still to try quoting the name or use `person` instead `user` as the name of the class.